st2common/st2common/persistence/cleanup.py

```diff
@@ -0,0 +1,77 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from st2common import config
+from st2common import log as logging
+from st2common.database_setup import db_config
+from st2common.models.db import db_cleanup as db_cleanup_func
```

Comment:
This method doesn't seem to exist

 /Volumes/workspace/src/storm/st2 ⮀ master ⮀ 9h2m ⮀ ag "db_cleanup_func"    ⮂
 /Volumes/workspace/src/storm/st2 ⮀ master ⮀ 9h2m ⮀

Reply:
I'm importing the `db_cleanup()` function from `st2common.models.db.__init__.py` as an alias `db_cleanup_func()`

Reply:
Ahh. I see it now. Sorry.